src/main/java/ox/app/game/Board.java

```diff
@@ -48,17 +53,35 @@ public void showBoard() {
     }
 
     private void changePrintMode(Symbol receivedSymbol, int iterator) {
```

Comment:
showing Board, changing print mode and all other methods connected with showing something to the console should be delegated to separate class. They're in Board class which is also handling board configuration. So Board class have multiple responsibilities